lib/charms/operator_libs_linux/v0/snap.py

```diff
@@ -260,24 +267,51 @@ def unset(self, key) -> str:
         """
         return self._snap("unset", [key])
 
-    def _install(self, channel: Optional[str] = "") -> None:
+    def _install(self, channel: Optional[str] = "", cohort: Optional[str] = "") -> None:
         """Add a snap to the system.
 
         Args:
+          confinemenet: strict or classic
```

Comment:
`confinemenet` => `confinement`

Also, this doesn't actually seem to be an argument to this function?